scripts/components/api-changes-validator/test-resources/test-projects/with-breaks/enum-break-remove-enum-member/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "enum-break-remove-enum-member",
```

Comment:
nit: let's add a `@aws-amplify` prefix to all the internal/external package, just in case